README.md

```diff
@@ -1,27 +1,89 @@
 
 ## Usage
 
-First, build your project with `tsc --generateTrace traceDir`.  This will create a new `traceDir` directory with paired trace and types files.
+The short version is to run these commands:
+
+```sh
+tsc -p tsconfig.json --generateTrace traceDir
+npm install --no-save @typescript/analyze-trace
+npx analyze-trace
+```
+
+Each of these commands do the following:
+
+1. First you will need to build your project with `--generateTrace` to a specific directory.
+   `tsc -p tsconfig.json --generateTrace traceDir` will create a new `traceDir` directory with paired trace and types files.
+   If your configuration file is not `tsconfig.json`, you will need to adjust the command.
+2. Installing `@typescript/analyze-trace` makes its various commands available in your project.
+3. Running `npx analyze-trace traceDir` outputs a sorted list of compilation hot-spots - places where TypeScript is taking a high amount of time.
 
-Next, install `@typescript/analyze-trace` to make its various commands available in your project.
+For best results, the analyzer should run on trace files that were generated in the same relative location.
+If the trace files fall out of date with your project, you may see different results.
+https://github.com/microsoft/typescript-analyze-trace/issues/29
+You can run `npx analyze-trace --help` to find out about other options including:
 
-For a sorted list of compilation hot-spots, run `npx analyze-trace traceDir`.
-Pass `--help` to learn more about configuration options.
-For best results, run the analyzer on a machine where the paths in the trace file(s) resolve correctly.
+Option                    | Default | Description
+--------------------------|---------|-------------------------------------------------------------------------------
+`--skipMillis [number]`   | `100`   | Suppress events that take less than the specified number of milliseocnds
+`--color [boolean]`       | `true`  | Color the output to make it easier to read.
+`--expandTypes [boolean]` | `true`  | Expand the names of types when printing them.
+`--json [boolean]`        | `false` | *Experimental and unstable*: Produce JSON output for programmatic consumption.
 
-For a simplified view of a types file (useful when investigating an individual trace), run `npx simplify-trace-types traceDir\types.json output_path`.
+For a simplified view of a `types.json` file (useful when investigating an individual trace), you can run `npx simplify-trace-types traceDir/types.json output.txt`.
 Note that the resulting file is for human consumption and should not be passed to the analyzer (i.e. don't clobber the original).
 
-To pretty-print individual types from a types file (faster than processing the entire file), run `npx print-trace-types traceDir\types.json id+`.
+### Interpreting Results
+
+The `analyze-trace` output will try to highlight the most expensive portions of a compilation that it was able to measure (a.k.a. "hot spots").
```

Comment:
thanks for defining `hot spots`.

Reply:
Do we want to say `npx analyze-trace` for consistency?